pkg/cloud/externalconn/connection.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package externalconn
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/cockroachdb/cockroach/pkg/cloud/externalconn/connectionpb"
+)
```

Comment:
that's a "mouthful"... maybe just proto dir?

Reply:
Are you suggesting renaming `connectionpb`? I just went with the nomenclature some other packages like `spanconfig` use.